code/game/objects/items.dm

```diff
@@ -337,6 +346,82 @@ GLOBAL_VAR_INIT(embedpocalypse, FALSE) // if true, all items will be able to emb
 	research_msg += "."
 	. += research_msg.Join()
 
+/**
+ *
+ * Details the stats of the examined weapon
+ *
+ * This function is called when the user clicks the hyperlink in
+ * /obj/item/examine. This function simply compiles some relevant
+ * weapon data and outputs it to the user.
+ * Arguments:
+ *  * href - Unused
+ *  * href-list - List provided by the href of input values, used to know what hyperlinked action is being attempted
+ */
+
+/obj/item/Topic(href, href_list)
+	if(href_list["examine"])
+		to_chat(usr, "<span class='notice'>[warning_label()]</span>")
+/**
+ *
+ * Compiles a warning label detailing various statistics of the examined weapon
+ *
+ * This function is called by the "examine" function of Topic(), and compiles a number of relevant
+ * weapon stats into a message that is then shown to the user
+ * Arguments:
+ *  * readout - Compiles all of the relevant text strings to be sent as a single message
+ *
+ */
+/obj/item/proc/warning_label(var/list/readout = "")
+	var/list/crimes = list("Assaults", "Third Degree Murders", "Robberies", "Terrorist Attacks", "Different Felonies", "Felinies", "Counts of Tax Evasion", "Mutinies")
+	var/list/victims = list("Human", "Moth", "Felinid", "Lizard", "Particularly Resilient Slime", "Syndicate Agent", "Clown", "Mortal Foe", "Innocent Bystander")
+	readout = list ("") /// Readout is used to store the text block output to the user so it all can be sent in one message
+
+	/// Meaningless flavor text
+	readout += "<span class='warning'>WARNING:</span> This item has been marked as dangerous by the NT legal team because of its use in <span class='warning'>[rand(2,99)] [crimes[rand(1, crimes.len)]].</span>"
```

Comment:
But the number of crimes (and type of crime) will change if you examine again.

It really should be fixed per item class. Doesn't have to be fixed per round, I guess.
(Ideally, you'd like, do something like temporarily seed the random number generator with
the item's typepath, and then these values would be deterministic, but that's awkward to
 do in DM).

Reply:
i would say either remove it or rework it into something meaningful. adding a meaningless number to this thing sounds fun at first, but it goes against the point of this feature. it would only confuse players and they will have to code-dive only to see that it's meaningless.

Reply:
If it were me I'd just have this number secretly be (I know I suggested this earlier) the sum of its stats. That way it has a useful gameplay purpose (indicates overall how powerful the item is) in a way that's intuitive (more uses = better weapon). I don't think the logic that the more powerful an item is the rarer and less likely it is to be used in a crime matters here.